core/themes/seven/template.php

```diff
@@ -149,3 +154,34 @@ function seven_breadcrumb($variables) {
 function seven_preprocess_maintenance_page(&$variables) {
   $variables['html_attributes']['class'][] = 'maintenance-page-wrapper';
 }
+
+/**
+ * Overridde theme_admin_block()
+ */
+function seven_admin_block($variables) {
+  $block = $variables['block'];
+  $output = '';
+
+  // Construct class
+  $path_array = explode('/', $variables['block']['path']);
+  $class_suffix = end($path_array);
```

Comment:
Seems like we could simply put this in `theme_admin_block()`, rather than overriding specifically in Seven.

Reply:
done.